src/app/modules/angular-slickgrid/styles/slick-plugins.scss

```diff
@@ -790,6 +790,21 @@ input.flatpickr.form-control {
     background-color: $flatpickr-bgcolor;
   }
 }
+.slickgrid-container {
+  .slick-headerrow {
+    .slick-headerrow-columns {
+      .slick-headerrow-column {
+        .flatpickr {
+          input {
```

Comment:
@gcko 
Thanks that is more organized this way. 
However, I found a small bug while recreating your PR in my other lib, there's a small issue with the Flatpickr styling, it shouldn't remove the left radius on the DateRange Filter (as can be seen below). 
To resolve this issue, you can add the `compound-input` class to the `input.compound-input`. Could you please apply the change and verify if that is all good on your side? You can test with [Example 25 - Range Filters](https://ghiscoding.github.io/Angular-Slickgrid/#/range)

![image](https://user-images.githubusercontent.com/643976/103501632-c967b580-4e1c-11eb-98e8-6c7a094900d5.png)

Reply:
@ghiscoding Good catch, the original intent is to ensure that compound filters that have a `.input-group-prepend` associated with Flatpickr do not border-left radius.

By adding `.compound-input` to `input`, it regresses the styling in this case:
![Screen Shot 2021-01-04 at 12 53 48](https://user-images.githubusercontent.com/521927/103502679-f3f15700-4e8c-11eb-9b58-1a71600d1da1.png)

However, by adding a sibling selector to `.flatpickr`, it allows for the removal of border radius specifically when there is a prepended filter:
![Screen Shot 2021-01-04 at 12 55 02](https://user-images.githubusercontent.com/521927/103502718-1a16f700-4e8d-11eb-9301-76f495d344ff.png)


```scss
...
.input-group-prepend + .flatpickr {
  input {
    border-top-left-radius: 0;
    border-bottom-left-radius: 0;
  }
}
...
```

Reply:
Addressed in https://github.com/ghiscoding/Angular-Slickgrid/pull/657/commits/91631a6acab308dce7ccf71c67001062b9618b1f

Reply:
That is why I asked you to verify, 2 pair of eyes are better than one 😉 
BTW, `.operator + .flatpickr` would have also work but I think the group prepend is probably better

Reply:
🤣  Let me know if there is anything else you see that needs work